pkgs/development/libraries/rlottie/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "rlottie";
+  version = "0.1";
+
+  src = fetchFromGitHub {
+    owner = "Samsung";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256:0fsh3imrl7di9076al1r43crw52q6ryrw642wgdv9rk0fmk3997h";
```

Comment:
Hash should use SRI-style value (base64-encoded hash separated by dash from method). I am surprised this even works.

Reply:
Replaced both occurrences.